packages/default/scss/scrollview/_layout.scss

```
@@ -14,164 +14,163 @@
-            padding: 0;
-            width: $scrollview-pagebutton-size;
-            height: $scrollview-pagebutton-size;
-            box-sizing: content-box;
+        & > li {
             display: inline-block;
-            position: relative;
-            flex: 0 0 $scrollview-pagebutton-size;
-            border-width: $scrollview-pager-item-border-width;
-            border-style: solid;
-            border-radius: 50%;
-            cursor: pointer;
-            pointer-events: all;
-
-            &::before {
-                content: "";
-                width: $scrollview-pagebutton-size + $scrollview-pager-item-spacing;
-                height: $scrollview-pager-height;
-                display: block;
-                position: absolute;
-                top: 50%;
-                left: 50%;
-                transform: translate(-50%, -50%);
-            }
+            overflow: hidden;
+            position: absolute;
+            top: 0;
+            left: 0;
         }
 
-        // remove when https://github.com/telerik/kendo-angular-scrollview/issues/130 is done
-        .k-scrollview-pageable > li.k-button::before {
-            pointer-events: initial;
-            border-radius: 0;
+        & > [data-role="page"] {
+            vertical-align: top;
+            display: inline-block;
+            min-height: 1px;
         }
+    }
 
-        .k-scrollview-next,
-        .k-scrollview-prev {
-            display: table;
-            position: absolute;
-            padding: 0;
-            height: 60%;
-            top: 20%;
-            text-decoration: none;
-            user-select: none;
-            cursor: pointer;
-            overflow: hidden;
-            -webkit-tap-highlight-color: $scrollview-arrow-tap-highlight-color;
-        }
+    kendo-scrollview.k-scrollview-wrap kendo-scrollview-pager,
+    .k-scrollview .k-scrollview-nav-wrap {
+
+        padding: 0;
+        margin: 0;
+        position: absolute;
+        left: 0;
+        right: 0;
+        bottom: $scrollview-pager-offset;
+        height: $scrollview-pager-height;
+        text-align: center;
+    }
 
-        .k-scrollview-prev {
-            left: 0;
+    kendo-scrollview.k-scrollview-wrap .k-scrollview-pageable,
+    .k-scrollview .k-scrollview-nav {
+        margin: 0;
+        padding:  $scrollview-pager-item-spacing;
+        max-width: 100%;
+        box-sizing: border-box;
+        line-height: 0;
+        text-align: center;
+        white-space: nowrap;
+        list-style: none;
+        display: inline-flex;
+        align-items: center;
+        overflow-x: scroll;
+        overflow-y: hidden;
+        pointer-events: initial;
+
+        &:focus,
+        &.k-state-focused {
+            outline: none;
         }
+    }
 
-        .k-scrollview-next {
-            right: 0;
+    kendo-scrollview.k-scrollview-wrap .k-scrollview-pageable > .k-button,
+    .k-scrollview .k-scrollview-nav > .k-link {
```

```suggestion
    .k-scrollview-pageable > .k-button,
    .k-scrollview-nav > .k-link {
```

use shorter selectors.